examples/rustc.html

```diff
@@ -20,13 +20,18 @@
 <body>
 <div id="terminal"></div>
 <div id="downloads"></div>
+<p id="note">
+  Note: the panic at the end is expected. This demo highlights how far `rustc` can get on this polyfill before failing due to other reasons.
+</p>
 <script type="module">
     import { Fd } from "../dist/fd.js";
     import { File, Directory } from "../dist/fs_core.js";
     import { PreopenDirectory } from "../dist/fs_fd.js";
-    import WASI from "../dist/wasi.js";
+    import { WASI, debug } from "../dist/index.js";
     import { strace } from "../dist/strace.js"
 
+    debug.enable();
```

Comment:
Maybe make this a method on WASI instances instead?

Reply:
SGTM. I didn't want to pollute the namespace. I can expose it as `WASI.debug` or have it handled with an `options` argument to the `WASI` constructor (e.g. `new WASI(args, env, fds, { debug: true })`). Or both.

The options arg would be useful if you foresee adding other configurable bits.